targets/TARGET_STM/TARGET_STM32WB/cmsis.h

```diff
@@ -14,10 +14,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 #ifndef MBED_CMSIS_H
 #define MBED_CMSIS_H
 
 #include "stm32wbxx.h"
 #include "cmsis_nvic.h"
 
+#include "stm32wbxx_ll_usart.h"
```

Comment:
does not seem the right place to include .. maybe to be checked if can be included from object.h ?